Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "pg_query"
 description = "PostgreSQL parser that uses the actual PostgreSQL server source to parse SQL queries and return the internal PostgreSQL parse tree."
-version = "0.8.2"
+version = "5.0.0"
```

Comment:
Why 5.0.0?

Reply:
To match the other libraries (pg_query and pg_query_go) -- they match the libpg_query versioning to clearly indicate each major Postgres release as its own major version. We could technically go to 1.0 here to do that, but I felt it'd be better to match the versions.